### PR TITLE
Grammar Fixes #2

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

This PR contains various grammar fixes for documentation.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/docs/products/ai/computer-vision/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    <p>
+        Jumping a bit forward, in 1966, a group within MIT's famous Project MAC (Project for
+        Mathematics and Computing) led by Seymour Papert set out to solve the computer vision
+        problem in a couple months during a summer camp <a href="#3">[3]</a>. Needless to say they
+        were extremely optimistic and unfortunately the project was a failure, however it was still
+        important as it is widely regarded as the birth of the field of computer vision in academic
+        study.
+    </p>
+
+    <p>
```
[the remark this answers is not in the document]
Comment (PineappleIOnic) [reply src/routes/docs/products/ai/computer-vision/+page.svelte]: We're splitting history into it's own blog post

File: src/routes/docs/products/ai/computer-vision/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+<Article
+    title="Computer Vision"
+    description="Learn about the basics of computer vision, the most popular tasks and applications of computer vision and how we can leverage Appwrite to build computer vision enabled applications."
+>
+    <p>
+        Computer Vision is a field of Artificial Intelligence where we work towards giving machines
+        a comprehensive understanding of visual data from a variety of sources, a few examples are:
+        Images, Videos, Point Clouds and X-Rays and MRI's from medical devices with the goal of
+        being able to further parse and process this information for downstream tasks.
+    </p>
```
[the remark this answers is not in the document]
Comment (PineappleIOnic) [reply src/routes/docs/products/ai/computer-vision/+page.svelte]: We're removing all the history stuff away from this page and we are splitting that into a blog post

File: src/routes/docs/products/ai/tutorials/object-detection/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+This code creates a new `AppwriteService` class that initializes the Appwrite client and provides a method to download a file from the Appwrite Storage.
+
+Now, we can import the class into the `src/index.js` file, at the top of the file, add the following line:
+```js
+import AppwriteService from './appwrite.js';
+```
+
+Then, we can use the `AppwriteService` class to download the image from the Appwrite Storage. After the bucket check add the following code:
+
+```js
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/object-detection/+page.markdoc:136]: I have no idea which file this is.

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+const hf = new HfInference(process.env.HUGGING_FACE_API_KEY);
+
+const result = await hf.imageClassification({
+  data: file,
+  model: 'microsoft/resnet-50',
+});
+```
+{% /section %}
+
+{% section #step-7 step=7 title="Create Database" %}
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:177]: Have this and the storage bucket config in one step call "configure Appwrite"

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+
+{% only_dark %}
+![Image Classification Test](/images/docs/ai/tutorials/image-classification/dark/result.png)
+{% /only_dark %}
+{% only_light %}
+![Image Classification Test](/images/docs/ai/tutorials/image-classification/result.png)
+{% /only_light %}
+{% /section %}
+{% section #step-12 step=12 title="Next steps" %}
+Congratulations! You've successfully created an image classification function using Appwrite and Hugging Face.
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:264]:
```suggestion
You've successfully created an image classification function using Appwrite and Hugging Face.
```

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+{% section #step-9 step=9 title="Deploy the function" %}
+With the function complete, we can now deploy it to Appwrite.
+{% partial file="deploy-function.md" /%}
+{% /section %}
+
+{% section #step-10 step=10 title="Configure Events" %}
+To test our function we're going to attach it directly to our Storage bucket using events.
+
+Navigate to your function in the Appwrite Console and visit it's **settings** tab,
+then under the **Events** section click on the **Add Event** button.
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:238]:
```suggestion
Navigate to your function in the Appwrite Console, under **Settings** > **Events**, click on the **Add Event** button.
```

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+{% /only_light %}
+{% /section %}
+{% section #step-12 step=12 title="Next steps" %}
+Congratulations! You've successfully created an image classification function using Appwrite and Hugging Face.
+
+From here you can expand on this project by adding more models or creating a front-end to upload images.
+We look forward to seeing what you build with this project! Full source code for this project can be found on
+[GitHub](https://github.com/appwrite/templates/tree/main/node/image-classification-with-huggingface).
+
+{% /section %}
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:270]: It's a little long. Can we combine some of the steps or skip some of them?

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Reviewed image classification

File: src/routes/docs/products/ai/tutorials/language-translation/+page.markdoc [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+```json
+{
+    "ok": true,
+    "output": "Bonjour, comment ça va?"
+}
+```
+
+{% /section %}
+{% section #step-12 step=12 title="Next Steps" %}
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/language-translation/+page.markdoc]: step 12?

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
@loks0n

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+```
+{% /section %}
+
+{% section #step-7 step=7 title="Create database" %}
+Before we can save our classification result to the Appwrite Database, we need to create a new database and collection in the Appwrite Console.
+
+Navigate to the Appwrite Console and click on **Database** in the left sidebar, then click on the **Create database** button, you can call this database anything you like, for this example we'll call it `AI`.
+Once you've created the database, click on the **Create collection** button and create a new collection, once again you can call it anything you want but for this example we'll call it `Image Labels`.
+
+Now we have a collection we're going to add two string attributes to it, `image` and `labels`. The `image` attribute will store the ID of the image we're classifying and the `labels` attribute will store the classification result.
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:188]:
```suggestion
Add two string attributes to the new collection, `image` and `labels`. The `image` attribute will store the ID of the image we're classifying and the `labels` attribute will store the classification result.
```

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 10]
```diff
@@ -0,0 +1,273 @@
...
+Both of these attributes should be `required` with `image` having the size of `256` and `labels` having the size of around `4096`.
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:189]: @PineappleIOnic Please use a table to show the attributes more cleanly

File: src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc [hunk 11]
```diff
@@ -0,0 +1,273 @@
...
+{% /only_dark %}
+{% only_light %}
+![Image Classification Database](/images/docs/ai/tutorials/image-classification/database.png)
+{% /only_light %}
+
+{% /section %}
+{% section #step-8 step=8 title="Save result" %}
+With the image classified, we can now save the result to the Appwrite Database.
+
+To begin with we're going to add a new function to the `appwrite.js` file we created earlier which will create these records in the database.
```
Comment (gewenyu99) [inline src/routes/docs/products/ai/tutorials/image-classification/+page.markdoc:202]:
```suggestion
To begin, add a new function to the `appwrite.js` file we created earlier which will create these records in the database.
```

**Review by gewenyu99 — DISMISSED** [review]

File: src/routes/docs/products/ai/tutorials/text-to-speech/+page.markdoc [hunk 12]
```diff
@@ -0,0 +1,431 @@
...
+
+{% section #step-2 step=2 title="Add Fetch" %}
+Once your function is created, navigate to the freshly created repository and either clone it to your local machine or launch a GitHub Codespace.
+
+Since we're going to be making requests to the ElevenLabs API and fetch isn't included in the current NodeJS 18 Runtime,
+we're going to install the `node-fetch` package.
+We're also going to install the `node-appwrite` package so we can upload the generated audio file to Appwrite Storage.
+
+```bash
+npm install node-fetch node-appwrite
```
[the remark this answers is not in the document]
Comment (PineappleIOnic) [reply src/routes/docs/products/ai/tutorials/text-to-speech/+page.markdoc:45]: Hugging Face API is borked when it comes to using TTS API's, either results in a timeout or the blob not being parsed correctly

**Review by loks0n — APPROVED** [review]

**Pull request merged**